src/entities/repos/submission_repo.py

```diff
@@ -91,19 +124,29 @@ async def upsert_filing(session: AsyncSession, filing: FilingDTO) -> FilingDAO:
 
 
 async def upsert_helper(session: AsyncSession, original_data: Any, type: T) -> T:
-    async with session.begin():
-        copy_data = original_data.__dict__.copy()
-        # this is only for if a DAO is passed in
-        # Should be DTOs, but hey, it's python
-        if copy_data["id"] is not None and "_sa_instance_state" in copy_data:
-            del copy_data["_sa_instance_state"]
-        new_dao = type(**copy_data)
-        new_dao = await session.merge(new_dao)
-        await session.commit()
-        return new_dao
-
-
-async def query_helper(session: AsyncSession, id: int, type: T) -> T:
-    async with session.begin():
-        stmt = select(type).filter(type.id == id)
-        return await session.scalar(stmt)
+    copy_data = original_data.__dict__.copy()
+    # this is only for if a DAO is passed in
+    # Should be DTOs, but hey, it's python
+    if copy_data["id"] is not None and "_sa_instance_state" in copy_data:
+        del copy_data["_sa_instance_state"]
+    new_dao = type(**copy_data)
```

Comment:
let's rename the T var from type to something else, type is a built-in python function.

Reply:
changed to table_obj